src/main/java/org/opentripplanner/transit/model/package.md

```diff
@@ -23,10 +23,10 @@ All transit entities must have an ID. Transit entities ar "root" level are consi
 roots_.
 
 
-#### @Nonnull and @Nullable entity fields
+#### Non-null and nullable entity fields
 
-All fields getters(except primitive types) should be annotated with `@Nullable` or `@Nonnull`. None 
-null field should be enforced in the Entity constructor by using `Objects.requireNonNull`, 
+All fields getters(except primitive types) should be annotated with `@Nullable` if they can return null.
```

Comment:
Maybe this line could be removed or it can just point to the codestyle since the codestyle document will now be more strict than what we are saying here.